### PR TITLE
TINY-3652: Cloud languages

### PR DESCRIPTION
**Pull request by lnewson**
Updated the cloud setup docs to mention that the languages can be loaded directly from the cloud now.

I wasn't sure if we should mention that only enterprise language packs are available? As the list available is really a subset of the community languages, so I figured if we did mention it, then we'd need to include a list which might be confusing or a bit too much for this page. So @tylerkelly13 (and anyone else who has an opinion) I wanted to get your take on what you think would be best?

### CODE REVIEW

**Comment by tylerkelly13** [comment]
> I wasn't sure if we should mention that only enterprise language packs are available?

Couldn't hurt, I think that would be helpful.

**Comment by lnewson** [comment]
> Couldn't hurt, I think that would be helpful.

Hmm, maybe on https://www.tiny.cloud/docs/configure/localization/#language we could add a new column to mention if it's available on cloud/enterprise?

**Comment by tylerkelly13** [comment]
> > Couldn't hurt, I think that would be helpful.
> 
> Hmm, maybe on https://www.tiny.cloud/docs/configure/localization/#language we could add a new column to mention if it's available on cloud/enterprise?

Sounds good :)

**Comment by lnewson** [comment]
> Sounds good :)

Done :)

File: _includes/configuration/language.md [hunk 1]
```diff
@@ -28,83 +28,83 @@ tinymce.init({
...
+| Bulgarian (Bulgaria)          | bg_BG   | {{ site.tick }}                                    |
+| Catalan                       | ca      | {{ site.tick }}                                    |
+| Chinese (China)               | zh_CN   | {{ site.tick }}                                    |
+| Chinese (Taiwan)              | zh_TW   | {{ site.tick }}                                    |
+| Croatian                      | hr      | {{ site.tick }}                                    |
+| Czech                         | cs      | {{ site.tick }}                                    |
+| Danish                        | da      | {{ site.tick }}                                    |
+| Divehi                        | dv      |                                                    |
+| Dutch                         | nl      | {{ site.tick }}                                    |
+| English (Canada)              | en_CA   |                                                    |
```
Comment (EkimChau) [inline _includes/configuration/language.md]: Should we add en_us for completeness?

Comment (lnewson) [reply _includes/configuration/language.md]: Maybe, though this is really talking about language packs you can download. `en_US` would be the default.

File: _includes/configuration/language.md [hunk 2]
```diff
@@ -28,83 +28,83 @@ tinymce.init({
...
-| Thai (Thailand)               | th_TH   |
-| Turkish                       | tr      |
-| Turkish (Turkey)              | tr_TR   |
-| Uighur                        | ug      |
-| Ukrainian                     | uk      |
-| Ukrainian (Ukraine)           | uk_UA   |
-| Vietnamese                    | vi      |
-| Vietnamese (Viet Nam)         | vi_VN   |
-| Welsh                         | cy      |
+| Language                      |  Code   | {{ site.cloudname }} /{{ site.newline }}Enterprise |
```
Comment (tylerkelly13) [inline _includes/configuration/language.md]: > {{ site.cloudname }} /{{ site.newline }}Enterprise
{{ site.cloudname }} /{{ site.newline }}{{site.enterpriseversion}}

File: cloud-deployment-guide/editor-and-features.md [hunk 3]
```diff
@@ -77,7 +77,10 @@ Ensure that the following URLs are accessible via this proxy if the network has
 Ensure the `tiny-api-key` and `tinymce-api-key` headers are retained while requesting the list of above URLs.
 
 ### Step 5: Specifying a translation
-[Download a language pack](https://www.tiny.cloud/get-tiny/language-packages/) to enable a language other than English (US). [Specify its location]({{ site.baseurl }}/configure/localization/#language_url) with the `language_url` configuration option.
+
+Extend the TinyMCE configuration to specify the [translation language]({{ site.baseurl }}/configure/localization/#language) with the `language` configuration option.
```
Comment (tylerkelly13) [inline cloud-deployment-guide/editor-and-features.md]: To add one or more translation languages, use the [language configuration option]({{ site.baseurl }}/configure/localization/#language).

Comment (lnewson) [reply cloud-deployment-guide/editor-and-features.md]: > To add one or more translation languages
That's incorrect info though, as you can only specify a single language.

File: _includes/configuration/language.md (hunk 2 above)
Comment (lnewson) [reply _includes/configuration/language.md]: Fixed

File: cloud-deployment-guide/editor-and-features.md (hunk 3 above)
Comment (lnewson) [reply cloud-deployment-guide/editor-and-features.md]: Fixed

**Pull request merged**